src/test/java/net/masterthought/cucumber/generators/integrations/PageTest.java

```diff
@@ -42,4 +46,12 @@ protected DocumentAssertion documentFrom(String pageName) {
             throw new ValidationException(e);
         }
     }
+
+    protected String[] getMessages(Output[] outputs) {
```

Comment:
ok, so part of this conversion is done in java and the rest in Velocity template. Consider make this code simpler